### ISSUE TITLE
remove 'tree' for OSX users from getting_started

### ISSUE THREAD

**Issue by janetriley** [railsbridge-boston/railsbridge-boston#22]
Step 5 of Creating A Rails Application (  views/curriculum/getting_started.markdown ) invites students to use 'tree' to see  all the files and directories that were created in their new suggestotron project. 

For OS X, it notes you may have to install the tree utility with homebrew.  Nobody had homebrew installed, and they got an error when they typed 'brew'.   This seems like a lot of work to show a minor point. 

I can think of three other ways to show the directories.

1) Use Finder.    From the terminal, the command 'open .'  opens the current directory in Finder.

2) Use SublimeText.  If the subl alias is installed,   'subl .'  .  From SublimeText 2, File > Open and choose the directory.   It adds a navigation area on the left, like the project view you get in IDEs.  Students didn't know they could do that, it's helpful to have the project view as a map  when the lectures talk about different parts of the project.  

3) For character-building, you can make it draw a text tree with 
ls -R | grep ":" | sed -e 's/://' -e 's/[^-][^\/]*\//--/g' -e 's/^/   /' -e 's/-/|/'
(Thanks, Google!)   

This is interesting, but not something that would be remembered and used later. Beginners should be in a committed relationship with the command line before experimenting with sed . 

I'd like to change the OSX instructions to 'open .' , and make a note about SublimeText for all platforms.  Thoughts?
-Janet

**Comment by bcarreno** [comment]
I agree with `change the OSX instructions to 'open .' , and make a note about SublimeText for all platforms`
If you make a PR I can merge it.

**Comment by slothbear** [comment]
I agree; students had trouble with `tree`.

The `open .` command looks good for Mac OS X.

I showed several people the Sublime Text navigation and they liked it. One person suggested including instructions for adding the `subl` symlink as they found it very handy.

I helped a student with Windows + Sublime Text open a folder. I think on Windows we had to use _File/Open Folder_. The plain _File/Open_ wouldn't let us open a folder.

**Comment by danchoi** [comment]
I agree with the above, let's get rid of the `tree` instructions and suggest  using `open .` on the Mac and whatever the equivalent is on Windows. On Ubuntu, `gnome-open .` may work. 

```
ls -R | grep ":" | sed -e 's/://' -e 's/[^-][^\/]*\//--/g' -e 's/^/ /' -e 's/-/|/'
```

is totally rad but definitely for a different crowd.

**Comment by danchoi** [comment]
Fixed by pull request 0ed4ca2
